### PR TITLE
Update misc.py

### PR DESCRIPTION
**Pull request by Remko76**
Added support for Wall Mounted Switch WRCC2

### CODE REVIEW

File: pyhomematic/devicetypes/misc.py [hunk 1]
```diff
@@ -137,7 +137,7 @@ def ELEMENT(self):
     "HM-PB-4Dis-WM-2": Remote,
     "HMW-IO-4-FM": Remote,
     "HMIP-WRC2": RemoteBatteryIP,
-    "HmIP-WRC2": RemoteBatteryIP,
```
Comment (danielperna84) [inline pyhomematic/devicetypes/misc.py:140]: Don't remove the support for HmIP-WRC2. 😉

Comment (Remko76) [reply pyhomematic/devicetypes/misc.py:140]: Aah...now I see that. I thought it was a double line. Didn't spot the difference between HMIP and HmIP...

You corrected it, I assume or do I need to do anything else?

Comment (danielperna84) [reply pyhomematic/devicetypes/misc.py:140]: No, you have to add it again on your branch and push the change. 👍

Comment (Remko76) [reply pyhomematic/devicetypes/misc.py:140]: Done....

File: pyhomematic/devicetypes/misc.py [hunk 2]
```diff
@@ -137,7 +137,8 @@ def ELEMENT(self):
...
+    "HMiP-WRC2": RemoteBatteryIP,
```
Comment (danielperna84) [inline pyhomematic/devicetypes/misc.py:140]: It's `HmIP-WRC2`, not `HMiP-WRC2`. 😁

**Comment by Remko76** [comment]
I am going to use the "I am new to this" excuse....but I do feel stupid with these mistakes... hope it is ok now;)

**Pull request merged**